packages/ajna-contracts/contracts/ajna/ajna-actions/AjnaProxyActions.sol

```diff
@@ -42,6 +42,20 @@ contract AjnaProxyActions {
         GUARD = _GUARD;
     }
 
+    /**
+     * @dev Emitted once an Operation has completed execution
+     * @param name Name of the operation
+     **/
+    event ProxyActionsOperation(bytes32 indexed name);
```

Comment:
I see that you are emitting strings where event has `bytes32`. Did you verify if events will have decodable names or hashes? For simplicity you might need to encode those strings to bytes32 before emitting event.